bridgesync/downloader.go

```diff
@@ -146,14 +146,13 @@ func buildClaimEventHandler(contract *polygonzkevmbridgev2.Polygonzkevmbridgev2,
 		claim := &Claim{
 			BlockNum:           b.Num,
 			BlockPos:           uint64(l.Index),
+			BlockTimestamp:     b.Timestamp,
+			TxHash:             l.TxHash,
 			GlobalIndex:        claimEvent.GlobalIndex,
 			OriginNetwork:      claimEvent.OriginNetwork,
 			OriginAddress:      claimEvent.OriginAddress,
 			DestinationAddress: claimEvent.DestinationAddress,
 			Amount:             claimEvent.Amount,
-			BlockTimestamp:     b.Timestamp,
-			TxHash:             l.TxHash,
-			FromAddress:        l.Address,
```

Comment:
why did we remove this?

Reply:
The `FromAddress` is set [here](https://github.com/agglayer/aggkit/blob/f8eb45673994c63e4b1c611d3e781c6411b2b702/bridgesync/processor.go#L256), in case full claims synchronization is enabled. It is misleading to set it to contract's address in case full claims sync is disabled. So IMO it is better to not set it, than set the misleading value.

Note that we are not setting [it](https://github.com/agglayer/aggkit/blob/f8eb45673994c63e4b1c611d3e781c6411b2b702/bridgesync/downloader.go#L146-L156) for Etrog claims either.